### ISSUE TITLE
Mandatory ssh_private_key_path and ssh_public_key_path when bastion_enabled is set to false

### ISSUE THREAD

**Issue by ferhatying** [oracle-terraform-modules/terraform-oci-oke#328]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version

```
Terraform v1.0.4
on darwin_amd64
Provider version 3.0.0
```

### Terraform Configuration Files

```hcl
module "oke" {
  source  = "oracle-terraform-modules/oke/oci"
  version = "3.3.0"

  compartment_id = var.compartment_id
  tenancy_id     = var.tenancy_ocid
  user_id        = var.user_ocid
  api_private_key_path = var.private_key_path
  api_fingerprint = var.fingerprint

  bastion_enabled = false
  use_encryption = false
  existing_key_id = ""
  ssh_public_key_path = ""

  label_prefix = var.label_prefix
  region       = var.location

  vcn_dns_label = module.vcn.vcn_all_attributes.dns_label
  vcn_name      = "oke-vcn-liftpoc"

  operator_timezone = "Europe/Amsterdam"

  metricserver_enabled   = true
  # vpa                    = true
  dashboard_enabled      = true
  create_service_account = true
}
```

### Expected Behavior

The arguments **ssh_private_key_path** and **ssh_public_key_path** should be optional when **bastion_enabled** is set to **false**. 

### Actual Behavior

```
╷
│ Error: Invalid function argument
│ 
│   on .terraform/modules/oke.base.operator/compute.tf line 35, in resource "oci_core_instance" "operator":
│   35:     ssh_authorized_keys = var.ssh_public_key != "" ? var.ssh_public_key : file(var.ssh_public_key_path)
│     ├────────────────
│     │ var.ssh_public_key_path is "none"
│ 
│ Invalid value for "path" parameter: no file exists at none; this function works only with files that are distributed as part of the configuration source code, so if
│ this file will be created by a resource in this configuration you must instead obtain this result from an attribute of that resource.
```

### Steps to Reproduce

1. `terraform apply`

**Comment by hyder** [comment]
Hi,

Thanks for logging this issue. 

We'll look to make the ssh keys optional and include this in the next patch release.

**Comment by hyder** [comment]
Hi @cosmindev and @ferhatying,

We've fixed this in #340.
